### DISCUSSION TITLE
Option to compare prices only (instead of update them)?

### DISCUSSION THREAD

**Discussion by hurt09** [rgriebl/brickstore#381] [Q&A]
Hi there!

Is there a functionality in BrickStore where I can only compare prices instead of updating them?

I'd like to load my inventory as XML (works fine already) and then display additional columns like "Price Guide Last 6 Months Sales" and "% Difference of current price" (or something like that).

I just want to check which prices are really outdated (like +/- 50%) in my inventory.

Is this possible somehow?

Thanks!

**Comment by ZZJHONS** [comment]
Did you try difference mode? it is under the view menu @hurt09

**Reply by hurt09** [reply]
Ah, even without that option, I saw that you can display columns "Pr.Orig" and "Pr.Diff".
Sorting by "Pr.Diff" after fetching prices for all items is a good start.
An additional column that displays percentage ("Pr. Orig" compared to "Price") would be really nice :-D
